.github/workflows/pep8.yml

```diff
@@ -0,0 +1,36 @@
+name: autopep8
+on: [push, pull_request]
+jobs:
+  autopep8_format:
+    runs-on: ubuntu-latest
+    if: ${{ github.event_name == 'push' }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: autopep8-format
+        id: autopep8
+        uses: peter-evans/autopep8@v1
+        with:
+          args: --recursive --in-place --aggressive --aggressive --exit-code --ignore E402 --max-line-length 120 .
```

Comment:
Do we really need the `--aggressive` here twice?

Reply:
It gives aggressiveness level 2 between [0-4]. Allows autpep8 to do some changes to solve errors in linting.

```
Use multiple --aggressive to increase the aggressiveness level. For example, E712 requires aggressiveness level 2 (since x == True could be changed to either x or x is True, but autopep8 chooses the former).
```